spec/features/movies_feature_spec.rb

```diff
@@ -0,0 +1,124 @@
+require "rails_helper"
+
+RSpec.describe "Movies", type: :feature do
+
+  before do
+    scorsese = create(:director, name: "Martin Scorsese", dob: "November 17, 1942")
+    departed = create(:movie, title: "The Departed", year: 2006, duration: 151, director: scorsese)
+    goodfellas = create(:movie, title: "Goodfellas", year: 1990, duration: 146, director: scorsese)
+
+    nolan = create(:director, name: "Christopher Nolan", dob: "July 30, 1970")
+    dark_knight = create(:movie, title: "The Dark Knight", year: 2008, duration: 152, director: nolan)
+    inception = create(:movie, title: "Inception", year: 2010, duration: 148, director: nolan)
+
+    leo = create(:actor, name: "Leonardo DiCaprio", dob: "November 11, 1974")
+    create(:character, name: 'Cobb', movie: inception, actor: leo)
+    create(:character, name: 'Billy Costigan', movie: departed, actor: leo)
+
+    jack = create(:actor, name: "Jack Nicholson", dob: "April 22, 1937")
+    create(:character, name: 'Frank Costello', movie: departed, actor: jack)
+
+    bob = create(:actor, name: "Robert De Niro", dob: "August 17, 1943")
+    create(:character, name: 'James Conway', movie: goodfellas, actor: bob)
+  end
+
+  context "index page" do
+    it "displays each movie's title and year", points: 5 do
+      visit "/movies"
+
+      movies = Movie.all
+      movies.each do |movie|
+        expect(page).to have_content(movie.title)
+        expect(page).to have_content(movie.year)
+      end
+    end
+
+    it "displays each movie's director", points: 10 do
+      visit "/movies"
+
+      movies = Movie.all
+      movies.each do |movie|
+        expect(page).to have_content(movie.director.name)
```

Comment:
instead of `expect(page).to have_content(movie.director.name)`,
would you prefer something like this?

```
director = Director.find_by(id: movie.director_id)
name = director.name
expect(page).to have_content(name)
```